addons/mail/static/src/chatter/web/chatter_patch.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    
+    <t t-inherit="mail.Chatter" t-inherit-mode="extension">
+        <xpath expr="//div[hasclass('o-mail-Chatter')]" position="attributes">
+            <attribute name="t-att-class">{ 'overflow-auto': props.isChatterAside, 'o-chatter-disabled': props.threadId === false }</attribute>
+        </xpath>
+        <xpath expr="//div[hasclass('o-mail-Chatter-top')]" position="replace">
```

Comment:
xpath with `replace` without `$0` somewhere are really not a good practice, and often indicate the code is not really designed correctly

Maybe the base chatter code should allow more entry points that are built in (registry or other ways) rather than having parts completely replaced.

Or maybe we should have 2 different templates completely. With each of them possibly including (t-call or sub-components) their common parts?

The portal template is so small that it might be worth just duplicating it.

Reply:
moved the whole `chatter.xml` the web folder